### ISSUE TITLE
Feature Request - Scrollbar in Run Summary

### ISSUE THREAD

**Issue by KOTR2003** [a85/POSTMan-Chrome-Extension#3097]
Postman Version: 4.10.7
App (Chrome app or Mac app): Windows
OS details: Windows 7 PRO SP1 64-bit

When using the runner directly in Postman (we have users here that need the GUI, as the Newman commands are too much for them), the 'Run Summary' does not seem to have a scrollbar. The only way to move left/right is the middle mouse button. It would be beneficial to have a scroll bar there as well I believe.

**Comment by prashantagarwal** [comment]
@KOTR2003 Thanks for reporting this. We have released a new version of Postman app 4.11.0 with the fix. Can you please update the app and verify.

**Comment by KOTR2003** [comment]
![image](https://cloud.githubusercontent.com/assets/13341993/26678810/478b3a8c-46a0-11e7-9cdf-15be1b6dfe7f.png)

I have updated to 4.11.0 and still do not see a scroll bar in the run summary.

**Comment by prashantagarwal** [comment]
@KOTR2003 Oh, We recently added a vertical scroll, I misread your issue. We will fix this in our upcoming releases.
Thanks again for bringing this up.

**Comment by Solksjaer** [comment]
Any updates on this issue?
I´m running Postman v6.7.3 version and there is no horizontal scrollbar for the case you use more than 40-50 iterations

**Comment by giridharvc7** [comment]
This has been fixed - Can you check if this is a problem you are still facing in 8.x?

**Comment by dctelus** [comment]
I am still facing this issue in v9.22.3; there is no way to scroll the run summary:
![image](https://user-images.githubusercontent.com/93535080/175368210-0b42ea6f-0b1a-4f39-8688-52c946547282.png)

**Comment by shubhbhargav** [comment]
Thanks for bringing this issue to our attention @dctelus. I am reopening it.

**Comment by nagu** [comment]
I am on version 10.18.6 on Mac and I still have this problem. 
UI Version: 10.18.6-ui-230922-0652

![Screenshot 2023-09-27 at 6 06 59 PM](https://github.com/postmanlabs/postman-app-support/assets/11967/4a401682-be48-4510-b5af-d1079dd8c98d)

**Comment by malvikach** [comment]
@nagu Thanks for your feedback. I checked this issue at my end and I am unable to reproduce this on a Mac machine using Postman 10.18.11 version. 

Here is a screen recording of navigating horizontally on the results using scroll. 


https://github.com/postmanlabs/postman-app-support/assets/89449769/04d3bfd4-eee2-4630-ab56-97b0a1e52d5c


Please take a look and let me know if you are still facing this issue. If yes- do provide us with a screen recording. 

Regards,
Malvika

**Comment by giridharvc7** [comment]
I'm closing this ticket now, please feel free to reopen this ticket if you face this issue!  Thank you & We appreciate your feedback!
